views/partials/route_segment_show_stats.html

```diff
@@ -0,0 +1,76 @@
+{{ define "route_segment_show_stats" }}
+<div id="routeSegmentChart"></div>
+
+<script>
+      (function () {
+     // Determine theme
+        let theme = "light";
+        if (
+          window.matchMedia &&
+          window.matchMedia("(prefers-color-scheme: dark)").matches
+        ) {
+          theme = "dark";
+        }
+
+     const rootElement = document.getElementById("routeSegmentChart");
+     const data = [
+     	{{ range .RouteSegmentMatches }}
+     		{{ if eq CurrentUser.ID .Workout.UserID  }}
+     			{ "x": new Date({{  .Workout.Date }}) ,  "y": {{ .AverageSpeed | HumanSpeed }} },
+     		{{ end }}
+     	{{ end }}
+     ];
+
+  const numberFormatter = new Intl.NumberFormat({ maximumFractionDigits: 2, minimumFractionDigits: 2 });
+
+     // Build chart
+     const options = {
+      theme: { mode: theme },
+      chart: {
+        height: 400,
+        animations: { enabled: false },
+        toolbar: { show: false },
```

Comment:
Remove this line to show the zoom controls - we should remove this from the other graphs too